user/src/main/resources/application.yml

```
@@ -66,3 +80,8 @@ management:
 kakao:
   api:
     key: ${KAKAO_MAP_KEY}
+
+logging:
+  pattern:
+    console: '%clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} [%-0.32X{traceId:-},%-0.16X{spanId:-}] %clr(${LOG_LEVEL_PATTERN:-%5p}) %clr(${PID:-}){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m%n${LOG_EXCEPTION_CONVERSION_WORD:%wEx}'
+    # ✅ 모든 콘솔 로그에 traceId, spanId 추가
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)